neo/Implementations/Wallets/peewee/UserWallet.py

```diff
@@ -525,8 +525,17 @@ def pretty_print(self, verbose=False):
                 'tokens': self._get_token_balances(addr_str)
             }})
 
+        aliases = dict()
+        alia = NamedAddress.select()
+        if len(alia):
+            for n in alia:
+                aliases[n.Title] = n.ToString()
+
         # pretty print
         for address, data in addresses.items():
+            for title, addr in aliases.items():
```

Comment:
Just as a general comment; 

I think we could have used `NamedAddress.get()` to query if an alias exists for the given address/scripthash instead of iterating over all available data. Now it doesn't really matter as we won't have a huge list of aliases. Just something to try and keep in mind when you encounter some performance critical part with larger data sets.

Reply:
Thank you for the feedback!